vms/event/services.py

```diff
@@ -105,6 +105,8 @@ def get_event_by_id(event_id):
 
     return result
 
+def get_event_by_name(event_name):
+    return Event.objects.get(name = event_name)
```

Comment:
@tonythomas01 .first() was not working here. I tried that but ended up getting no result.

Reply:
That is probably due to an Event with that name do not exist ? 

(even though IMHO - why are you not using the Event name and not Event.id) ? What if there are two events with same name ?

Reply:
@tonythomas01 I checked modals and found that name is unique field for event. Also dropdown contains only names so what i get from dropdown is only event name.

Reply:
Well, you can add the Event.id to the dropdown item id and get it in the JS (if you want). 

But - given the name is unique - I guess it should be okey (even though not-recommended as string parsing is a mess always). 

 Also, see that  you cannot just use  `Event.objects.get(name=event_name)` as it would throw an `Event.DoesNotExist`. If it works, then `Event.objects.filter(name=event_name).first()` should work.